src/app.ts

```diff
@@ -3,7 +3,7 @@ import * as bodyParser from "body-parser";
 import * as crypto from 'crypto';
 import express from "express";
 import { Request, Response } from "express";
-import { RepeatingInvoices, Schedule, TokenSetParameters } from 'xero-node'
+import { BatchPaymentDelete, BatchPaymentDeleteByUrlParam, HistoryRecord, RepeatingInvoices, Schedule, TokenSetParameters } from 'xero-node'
```

Comment:
Anything from the Accounting API, please import from xero-node below. Notice that anything from the other API sets have to be imported from their respective models